scripts/install_requirements.sh

```diff
@@ -3,4 +3,5 @@
 pip install -r requirements.txt
 python -m nltk.downloader punkt
 python -m spacy.en.download all
-pip install --no-cache-dir http://download.pytorch.org/whl/cu80/torch-0.1.11.post5-cp35-cp35m-linux_x86_64.whl
\ No newline at end of file
+pip install --no-cache-dir http://download.pytorch.org/whl/cu80/torch-0.1.11.post5-cp35-cp35m-linux_x86_64.whl
+conda install pytorch torchvision -c soumith
```

Comment:
@DeNeutoy I had to run this command again on my work mac (yesterday I was using a home mac) in order for the tests to pass.